core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnPredictionOpDesc.scala

```diff
@@ -17,6 +17,15 @@ class SklearnPredictionOpDesc extends PythonOperatorDescriptor {
   @JsonPropertyDescription("attribute name of the prediction result")
   var resultAttribute: String = _
 
+  @JsonProperty(
+    value = "Ground Truth Attribute Name to Ignore",
+    required = false,
+    defaultValue = ""
+  )
+  @JsonPropertyDescription("attribute name of the ground truth")
+  @AutofillAttributeName
+  var groundTruthAttribute: String = ""
```

Comment:
val?

Reply:
I think it is var because this is a variable that user selects.